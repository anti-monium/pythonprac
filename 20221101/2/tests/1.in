A = Triangle((1,5), (-1,3), (-3,1))
B = Triangle((-2,-1), (-4,-3), (-2,-4))
C = Triangle((2,3), (3,-3), (-2,-1))
print(round(abs(A), 3), round(abs(B), 3), round(abs(C), 3))
if bool(A):
	print('A exists')
else:
	print('A doesn\'t exist')
if bool(B):
	print('B exists')
print(A in B, A in C, B in C, C in B)
print(A & B, A & C, B & C)
if A < C:
	print(100)
else:
	print(500)
if B < A:
	print(100500)
else:
	print('Max')
