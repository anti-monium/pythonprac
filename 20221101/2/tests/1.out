0 3.0 14.0
A doesn't exist
B exists
True True False False
False False True
100
Max
